.github/workflows/nia-e2e.yaml

```diff
@@ -0,0 +1,49 @@
+name: Now in Android E2E
+
+on:
+  workflow_dispatch:
+  pull_request:
+    types: [opened, reopened, synchronize, labeled]
+    branches:
+      - main
+    paths:
+      - '.github/workflows/nia-*'
+      - 'nowinandroid/**'
+
+permissions:
+  contents: read
+  
+jobs:
+  nia-e2e:
+    name: Setup Anbox and Run E2E Tests
+    runs-on: ubuntu-latest
+    if: contains(github.event.pull_request.labels.*.name, 'run-nia-e2e-tests')
+    timeout-minutes: 60
+    defaults:
+      run:
+        working-directory: ./nowinandroid
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@08c6903cd8c0fde910a37f88322edcfb5dd907a8 # v5.0.0
+    
+    - name: Setup Android Dependencies
+      uses: ./.github/actions/setup-android
```

Comment:
I am not sure how this slipped through but the authorship for the uses https://github.com/android-actions is not clear to me and we should refrain for the example from using any 3rd party actions other than what comes from Github (https://github.com/actions). As also discussed we can assume and require GitHub runner compatible images which already should come with everything we need. Was there a specific reason for which you had to add this?